### PR TITLE
Update for new build API

### PR DESCRIPTION
**Pull request by linusg**

### CODE REVIEW

File: args.zig [hunk 1]
```diff
@@ -475,7 +475,7 @@ fn convertArgumentValue(comptime T: type, allocator: std.mem.Allocator, textInpu
                 // This is equivalent to allocator.dupeZ but works with any sentinel.
                 if (comptime std.meta.sentinel(T)) |sentinel| {
                     const data = try allocator.alloc(u8, textInput.len + 1);
-                    std.mem.copy(u8, data, textInput);
+                    @memcpy(data, textInput);
```
Comment (MasterQ32) [inline args.zig:478]: Will assert that `data.len == textInput.len` which is not given. You should slice data here

File: args.zig [hunk 2]
```diff
@@ -1031,15 +1029,10 @@ test "full help" {
...
-        pub const meta = .{
-            .name = "test",
-            .full_text = "testing tool",
-            .usage_summary = "[--boolflag] [--stringflag]",
-            .option_docs = .{
-                .boolflag = "a boolean flag",
-                .stringflag = "a string flag",
-            }
-        };
+        pub const meta = .{ .name = "test", .full_text = "testing tool", .usage_summary = "[--boolflag] [--stringflag]", .option_docs = .{
```
Comment (MasterQ32) [inline args.zig:1032]: Here's a missing comma at the end

File: args.zig [hunk 3]
```diff
@@ -1070,13 +1063,10 @@ test "help with no usage summary" {
...
         };
 
-        pub const meta = .{
-            .full_text = "testing tool",
-            .option_docs = .{
-                .boolflag = "a boolean flag",
-                .stringflag = "a string flag",
-            }
-        };
+        pub const meta = .{ .full_text = "testing tool", .option_docs = .{
```
Comment (MasterQ32) [inline args.zig:1066]: Same comma here

**Review by MasterQ32 — CHANGES_REQUESTED** [review]

**Comment by linusg** [comment]
Done :)

**Pull request merged**